corehq/couchapps/last_modified/views/by_last_modified/map.js

```diff
@@ -1,7 +1,12 @@
 function(doc){
-    if (doc.doc_type == "Group") {
-        emit(doc.last_modified, null);
-    } else if (doc.doc_type == "Domain") {
-        emit(doc.last_modified, null);
+    if (doc.doc_type == "Group" || doc.doc_type == "Group-Deleted") {
+        emit([doc.doc_type, doc.last_modified], null);
+    } else if (doc.doc_type == "Domain" || doc.doc_type == "Domain-Deleted") {
+        emit([doc.doc_type, doc.last_modified], null);
+    } else if (doc.doc_type == "WebUser" ||
+               doc.doc_type == "WebUser-Deleted" ||
+               doc.doc_type == "CommCareUser" ||
+               doc.doc_type == "CommCareUser-Deleted") {
```

Comment:
actually we'll have to use base doc, at least for the deleted suffix: https://github.com/dimagi/commcare-hq/blob/8063cd3ff1f889d58a997c00557be092bda8e214/corehq/apps/users/models.py#L1611